### PR TITLE
Fix recurring spelling [ci skip]

### PR DESCRIPTION
**Pull request by sonalkr132**
~~Remove trailing white space.
Remove duplicate contact section from contribution.md (it already
exists on readme.md)~~
Fix spelling.

~~[Contact section of reame](https://github.com/openSUSE/osem#contact)~~
~~> GitHub issues are the primary way for communicating about specific proposed changes to this project. If you have other questions feel free to subscribe to the opensuse-web@opensuse.org mailinglist, all OSEM contributors are on that list! Additionally you can use #osem channel on freenode IRC.~~

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -86,9 +86,6 @@ vagrant exec bundle exec rspec
 ## Code of Conduct
 OSEM is part of the openSUSE project. We follow all the [openSUSE Guiding Principles!](http://en.opensuse.org/openSUSE:Guiding_principles) If you think someone doesn't do that, please let us know at maintainers@osem.io
 
-## Contact
```
Comment (hennevogel) [inline CONTRIBUTING.md]: Why remove this?

Comment (sonalkr132) [inline CONTRIBUTING.md]: In f188fd55 this section was added to REAME.md. Now it is just a duplicate information(with exact same words).

If we are going to keep contact section in contribution.md, can we please use different words?

Comment (hennevogel) [inline CONTRIBUTING.md]: Why? Are we in the word saving business?

Comment (sonalkr132) [inline CONTRIBUTING.md]: I have added it back.

**Pull request merged**